addon/components/bs-list-group/item.js

```diff
@@ -0,0 +1,80 @@
+import Component from '@glimmer/component';
+import arg from '../../utils/decorators/arg';
+
+/**
+ * Component to create the items for [List Group](Components.ListGroup.html).
+ *
+ * @class ListGroupItem
+ * @namespace Components
+ * @extends Glimmer.Component
+ * @public
+ */
+export default class BsListGroupItemComponent extends Component {
+  /**
+   * Tag for List group item component.
+   *
+   * Setting its value to `a` will automatically add the attribute `href="#"`.
+   *
+   * Setting its value to `button` will automatically add the attribute `type="button"`.
+   *
+   * This default attributes can be overridden.
+   *
+   * @default 'li'
+   * @type string
+   * @public
+   */
+  @arg
+  htmlTag = 'li';
```

Comment:
Do we need to make the user specify this? Or can't we make this explicit, to be e.g. `button` when `@action={{true}}`, `li` otherwise? Seems for actionable items we force the user to set both, which seems redundant. It plain Bootstrap that's how it is with static markup, we can apply smartness into our component here, right?

Also, is there any good reason to use an `<a>` instead of a `<button>`? I feel if the link has no real `href`, it shouldn't be a link (e.g. a11y-wise)?